src/track/trackinfo.h

```diff
@@ -34,6 +34,9 @@ class TrackInfo final {
 #if defined(__EXTRA_METADATA__)
     MIXXX_DECL_PROPERTY(QString, language, Language)
     MIXXX_DECL_PROPERTY(QString, lyricist, Lyricist)
+#endif // __EXTRA_METADATA__
+    MIXXX_DECL_PROPERTY(QString, lyrics, Lyrics)
```

Comment:
This change is probably also worth a separate PR.

Reply:
This is mostly a patch for my personal requirements but I'm free to contribute it upstream too.